packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -201,6 +202,21 @@ function Tooltip(props) {
     }
   };
 
+  const getOwnerDocument = React.useCallback(() => {
+    if (childNode == null) {
+      return null;
+    }
+    return childNode.ownerDocument;
```

Comment:
What about the `ownerDocument.js` utils, should we use it?

Reply:
Should this logic be abstracted (moved) in `useIsFocusVisible()`? I can imagine that we will duplicate it otherwise the day we use `useIsFocusVisible()` in another component.

Reply:
We will see.

Reply:
The utility function looks too defensive. It shouldn't silently use another document.